src/common.cpp

```diff
@@ -55,11 +55,16 @@ std::string sockaddr_to_hostport(const struct sockaddr *addr)
     case AF_INET6:
         addrlen = sizeof(sockaddr_in6);
         break;
+    case AF_UNIX:
+        addrlen = sizeof(sockaddr_un);
+        break;
     }
     getnameinfo(addr, addrlen, hbuf, sizeof(hbuf), pbuf, sizeof(pbuf),
             NI_NUMERICHOST | NI_NUMERICSERV);
     if (addr->sa_family == AF_INET6) {
         return "[" + std::string(hbuf) + "]:" + std::string(pbuf);
+    } else if (addr->sa_family == AF_UNIX ) {
+        return std::string(pbuf) + " [UDS]";
```

Comment:
should not it be `std::string(hbuf)` only
see
man for `getnameinfo`
```
If	a link-layer address or	UNIX-domain address is passed to
     getnameinfo(), its	ASCII representation will be stored in host.  The
     string pointed to by serv will be set to the empty	string if non-NULL;
     flags will	always be ignored.
```

Reply:
Not sure if I understand, currently I use 'pbuf' and it returns the path correctly
![pbuf](https://user-images.githubusercontent.com/44922030/165500906-315eac18-3293-4566-98e9-a895b6cd4ea5.JPG)

Reply:
when using hbuf I see
![hbuf](https://user-images.githubusercontent.com/44922030/165503664-b5cddf3c-2975-43ef-a397-0bcd278675f1.JPG)

Reply:
ok